lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
I had started at one point to go into the remaining File calls and redirect them through `Puppet::FileSystem` - I wonder if that should be investigated agai.